crates/core/src/factories/core_fetch.rs

```diff
@@ -106,15 +107,19 @@ impl FetchFactory for CoreFetchFactory {
 pub struct CoreFetchConfig {
     /// How many parallel op fetch requests can be made at once. Default: 2.  
     pub parallel_request_count: u8,
-    /// Duration in ms to keep an unresponsive agent on the cool-down list. Default: 120_000.
-    pub cool_down_interval_ms: u64,
+    /// Duration in ms to keep an unresponsive agent on the back off list. Default: 120_000.
+    pub back_off_interval_ms: u64,
+    /// Maximum exponent for back off interval. Back off duration is calculated
+    /// back_off_interval * 2^back_off_exponent. Default: 4.
+    pub max_back_off_exponent: u32,
```

Comment:
That gives a maximum back-off of 32 minutes right? I wonder if there's a library we could use here because this a fiddly thing to implement

Ideally we'd back off by a few seconds and probably not by more than say 5 minutes

Reply:
Changed default to 20 sec with exp 4. That gives us 20 sec initially and goes up to 11:40 min.